src/Spice86.Core/Emulator/Devices/Sound/PCSpeaker/PitMode.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Spice86.Core.Emulator.Devices.Sound.PCSpeaker;
+/// <summary>
+/// PIT operation modes (for the PC Speaker)
+/// </summary>
+internal enum PitMode : byte {
```

Comment:
Shouldn't this be part of the pit instead?
Don't we have such definition there already?

Reply:
It is part of the PIT now.